pkg/apiserver/statement/statement.go

```diff
@@ -122,6 +123,7 @@ type GetStatementsRequest struct {
 	BeginTime int      `json:"begin_time" form:"begin_time"`
 	EndTime   int      `json:"end_time" form:"end_time"`
 	Text      string   `json:"text" form:"text"`
+	Fields    string   `json:"fields" form:"fields"`
```

Comment:
If I remember correctly, it is safe to be just:

```go
Fields string[] `json:"fields" form:"fields"
```

Our clients will work fine!

Reply:
I tried this, this will make the URL much longer, for example, `fields=schema_name,digest,sum_latency` will become `fields[]=schema_name&fields[]=digest&fields[]=sum_latency`, I am a little afraid that will beyond the URL length limitation.

Reply:
Good point! Then let's keep the current way.